rustls/tests/common/mod.rs

```diff
@@ -28,10 +29,14 @@ use rustls::{
     DigitallySignedStruct, DistinguishedName, Error, InconsistentKeys, NamedGroup, ProtocolVersion,
     RootCertStore, ServerConfig, ServerConnection, SideData, SignatureScheme, SupportedCipherSuite,
 };
+
 use webpki::anchor_from_trusted_cert;
 
 use super::provider;
 
+// Import `Arc` here for tests - can be overwritten to test with another Arc  such as `portable_atomic_util::Arc`
```

Comment:
nits: consistent backtick usage, extra space between "Arc" and "such":

```suggestion
// Import `Arc` here for tests - can be overwritten to test with another `Arc` such as `portable_atomic_util::Arc`
```